### PR TITLE
Sniff for required attributes of QueryParam annotation added

### PR DESCRIPTION
**Pull request by asev**
All QueryParam annotations must have `description` and `strict` attributes.

### CODE REVIEW

File: src/InterNations/Sniffs/Syntax/QueryParamSniff.php [hunk 1]
```diff
@@ -0,0 +1,48 @@
+<?php
+namespace InterNations\Sniffs\Syntax;
+
+use PHP_CodeSniffer_File as CodeSnifferFile;
+use PHP_CodeSniffer_Sniff as CodeSnifferSniff;
+
+class QueryParamSniff implements CodeSnifferSniff
+{
+    private static $requiredAttributes = [
```
Comment (lstrojny) [inline src/InterNations/Sniffs/Syntax/QueryParamSniff.php]: Can be a constant

File: src/InterNations/Sniffs/Syntax/QueryParamSniff.php [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    ];
+
+    public function register()
+    {
+        return [T_DOC_COMMENT];
+    }
+
+    public function process(CodeSnifferFile $file, $stackPtr)
+    {
+        $regex = '/\* @.*QueryParam\(.*/';
```
Comment (lstrojny) [inline src/InterNations/Sniffs/Syntax/QueryParamSniff.php]: A slightly less aggressive regex:
```php
'/\* @[^\(]*QueryParam\(.*/';
```

**Comment by lstrojny** [comment]
Nice sniff and extra props for the tests! :+1:

File: src/InterNations/Sniffs/Syntax/QueryParamSniff.php (hunk 1 above)
Comment (asev) [inline src/InterNations/Sniffs/Syntax/QueryParamSniff.php]: Array constants are supported from PHP 5.6. And there are travis builds for 5.4 and 5.5, that will fail.

Comment (lstrojny) [inline src/InterNations/Sniffs/Syntax/QueryParamSniff.php]: Good point!

**Pull request merged**